game/src/Game/Autoload/CursorManager.gd

```diff
@@ -0,0 +1,173 @@
+extends Node
+
+class CompatCursor:
+	#cursor properties
+	var cursor_name : String
+
+	var resolutions : PackedVector2Array
+	var frames : Array[ImageTexture]
+	var hotspots : PackedVector2Array
+	var is_animated : bool = false
+	var sequence : PackedInt32Array = [0]
+	var timings : PackedFloat32Array = [1.0]
+	
+	#Cursor state
+	var currentFrame : int = 0
+	var timeToFrame : float = 1.0
+	
+	func _init(nameIn:String) -> void:
+		self.cursor_name = nameIn
```

Comment:
Not a big deal, but I believe you can remove all uses from `self.` so long as there aren't any variables with conflicting names, e.g. function arguments